docs/docs/using-client-side-only-packages.md

```diff
@@ -80,6 +80,29 @@ const LoadableBuyButton = Loadable(() => import("./ShopifyBuyButton"))
 export default LoadableBuyButton
 ```
 
+## Workaround 4: Use React.lazy and Suspense on client side only
+
+React.lazy and Suspense are still not ready for server side rendering but they can still be used by checking if we are doing server side rendering.
+
+```jsx
+const MyPage = () => {
+  const isSSR = typeof window === "undefined"
+  const ClientSideOnlyLazy = isSSR
```

Comment:
It is never used when `isSSR == true` 
so you don't need this here?

Reply:
You're right. I didn't notice it's redundant.
I'll test it anyway and do the change.

Reply:
Yes, of course it works :) I just wasn't really sure what `React.lazy` is doing to webpack.
Docs updated.